single-line-diagram/single-line-diagram-cgmes/single-line-diagram-cgmes-layout/src/main/java/com/powsybl/sld/cgmes/layout/LayoutToCgmesDlExporterTool.java

```diff
@@ -118,7 +118,7 @@ public void run(CommandLine line, ToolRunningContext context) {
         Network network = Network.read(inputFile);
 
         context.getOutputStream().println("Generating layout for the network ...");
-        LayoutToCgmesExtensionsConverter lTranslator = new LayoutToCgmesExtensionsConverter(sFactory, vFactory, new LayoutParameters().setUseName(true));
+        LayoutToCgmesExtensionsConverter lTranslator = new LayoutToCgmesExtensionsConverter(sFactory, vFactory, new LayoutParameters().setCgmesUseNames(true));
```

Comment:
Why cgmesUseNames ? ID vs name is an IIDM concept, not only related to CGMES.

Reply:
It is a workaround I found in order not to touch this part of the code in this PR. I created an https://github.com/powsybl/powsybl-diagram/issues/510 to track that.